src/lib/PushUtils.js

```diff
@@ -307,6 +307,10 @@ export function tableInfoBuilder(query, schema, styles = {}) {
     return;
   }
 
+  try {
```

Comment:
Yeah, that makes sense. 
I had experienced some strange behaviour on different Parse Server apps where sometimes `query` was a JSON string and other times it was already an object. Nevertheless, this should be okay.